libs/hook-dapp-lib/src/hookDappsRegistry.json

```diff
@@ -49,5 +49,19 @@
     "conditions": {
       "supportedNetworks": [11155111]
     }
+  },
+  "CLAIM_VESTING": {
+    "id": "177eda1e6d1d89f1777d60387f06e1fdcb2c11baad38c6f06c8a3404ebc7e03a",
+    "type": "IFRAME",
+    "name": "Claim Vesting Hook",
+    "description": "The Claim Vesting Hook is a powerful and user-friendly feature designed to streamline the process of claiming funds from LlamaPay vesting contracts. This tool empowers users to easily access and manage their vested tokens, ensuring a smooth and efficient experience in handling time-locked assets.",
+    "descriptionShort": "Claim your LlamaPay vesting contract funds",
```

Comment:
Is this exclusive to LlamaPay?

If that's the case, I suggest to name it so to be explicit and not set wrong expectations.